<launch>
    <arg name="model" default="$(find rwth_ground_hog)/model/config" />
    <arg name="ground_plane" default="/ground_plane" />
    <arg name="queue_size" default="10" />
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="detections" default="/groundHOG/detections" />
    <arg name="result_image" default="/groundHOG/image" />
    <arg name="detected_persons" default="/detected_persons"/>
    <arg name="detection_id_offset" default="0"/>
    <arg name="detection_id_increment" default="1"/>
    <arg name="world_scale" default="1"/>
    <arg name="score_thresh" default="0.0"/>

    <node pkg="rwth_ground_hog" type="groundHOG" name="groundHOG" output="screen">
        <param name="world_scale" value="$(arg world_scale)"/>
	<param name="score_thresh" value="$(arg score_thresh)"/>
        <param name="model" value="$(arg model)" type="string"/>
        <param name="ground_plane" value="$(arg ground_plane)" type="string"/>
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="detections" value="$(arg detections)" type="string"/>
        <param name="result_image" value="$(arg result_image)" type="string"/>
        <param name="detected_persons" value="$(arg detected_persons)" type="string"/>
        <param name="detection_id_offset" value="$(arg detection_id_offset)"/>
        <param name="detection_id_increment" value="$(arg detection_id_increment)"/>
    </node>

</launch> 
